src/patternfly/components/Form/form.scss

```diff
@@ -188,6 +192,20 @@
   --pf-c-form__helper-text--MarginTop: 0;
 
   padding-bottom: var(--pf-c-form__group-label--PaddingBottom);
+
+  &.pf-m-info {
+    display: flex;
+    align-items: flex-end;
```

Comment:
```suggestion
```

Reply:
Don't think you want this..

![Screen Shot 2021-06-30 at 2 37 51 PM](https://user-images.githubusercontent.com/5385435/124013891-df5fc880-d9b0-11eb-8879-36e52c9113c4.png)

Reply:
Isn't that what @mcoker suggested to add so that info is right above the field rather than floating at the top?

Reply:
Disregard, looks like you already considered this above ^^